encoding/protobuf/error_integration_test.go

```diff
@@ -185,15 +187,17 @@ func TestProtoGrpcStreamServerErrorDetails(t *testing.T) {
 	assert.Equal(t, errorMsg, st.Message(), "unexpected error message")
 
 	actualDetails := protobuf.GetErrorDetails(err)
-	assert.Equal(t, expectedDetails, actualDetails, "unexpected error details")
+	if !proto.Equal(expectedDetails[0].(proto.Message), actualDetails[0].(proto.Message)) {
+		t.Errorf("unexpected error details")
+	}
```

Comment:
This should be `assert.True(t, proto.Equal(expectedDetails[0].(proto.Message), actualDetails[0].(proto.Message))`

Why did you need to change this test?